python/cac_tripplanner/destinations/views.py

```diff
@@ -130,3 +133,24 @@ def get(self, request, *args, **kwargs):
             results = results[:limit_int]
         data = serializers.serialize('json', results, fields=output_fields)
         return HttpResponse(data, 'application/json')
+
+class FeedEvents(View):
+    """ API endpoint for the FeedEvent model """
+
+    def get(self, request, *args, **kwargs):
+        """ GET 20 most recent feed events that are published
+
+        TODO: Additional filtering, dynamic limits?
```

Comment:
I also limited to 20 events because there is no end date in the feed. Only the publish date. The event date is occasionally buried in the content, which would be way too much effort to parse I think.